src/startupscriptgenerator/dotnetcore/scriptgenerator.go

```diff
@@ -102,51 +104,43 @@ func (gen *DotnetCoreStartupScriptGenerator) getStartupCommand() (string, string
 	// as we want to generate a script which does a 'cd' to this directory and run the user startup command
 	publishOutputDir := gen.PublishedOutputPath
 	if publishOutputDir == "" {
-		projDetails := gen.getProjectDetailsAndCache()
-		if projDetails.FullPath == "" {
-			return "", ""
-		}
 
-		publishOutputDir = filepath.Join(projDetails.Directory, OryxPublishOutputDirectory)
+		// Check if current directory is indeed the output directory (Example, like in AppService's case)
+		currentDir := common.GetValidatedFullPath(".")
```

Comment:
>currentDir := common.GetValidatedFullPath(".") [](start = 2, length = 46)

what path is this supposed to return? The path from where the command was executed from?

Reply:
yes, right